unlock-app/src/components/interface/GlobalErrorConsumer.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { GlobalErrorContext } from '../../utils/GlobalErrorProvider'
+import { mapping } from '../creator/FatalError'
+import Layout from './Layout'
+
+const Consumer = GlobalErrorContext.Consumer
+
+export const displayError = (error, children) => {
+  if (error) return <Layout title="">{error}</Layout>
```

Comment:
Do you mind putting the return statement on a new line with `{}`? I think this would be more consistence with out style guide (I thought there was an eslint rule for that?)

Reply:
oops saw this post-merge, I'll do it in a separate PR